docs/tech-reference/iso-rsync-distributor.rst

```diff
@@ -12,10 +12,15 @@ Configuration
 Pulp's SELinux policy includes a ``pulp_manage_rsync`` boolean. When enabled, the
 ``pulp_manage_rsync`` boolean allows Pulp to use rsync and make ssh connections. The boolean is
 disabled by default. The ISO Rsync distributor will fail to publish with SELinux Enforcing unless
-the boolean is enabled. To enable it, you can do this::
+the boolean is enabled. To enable it by default, you can do this::
 
     $ sudo semanage boolean --modify --on pulp_manage_rsync
 
+In F27+ `semanage` only sets the default state of an SELinux policy, an additional command must also
+be run to set its current state::
+
+    $ sudo setsebool pulp_manage_rsync on
```

Comment:
I think that we can consolidate this back into one command.  ```setsebool``` has a "persist" option which we can use to set the default on the policy across reboots.

    $ sudo setsebool -P pulp_manage_rsync on